src/user-event/press/__tests__/longPress.real-timers.test.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { Pressable, Text } from 'react-native';
+import { render, screen } from '../../../pure';
+import { userEvent } from '../..';
+import * as WarnAboutRealTimers from '../utils/warnAboutRealTimers';
+
+const mockWarnAboutRealTimers = jest
+  .spyOn(WarnAboutRealTimers, 'warnAboutRealTimers')
+  .mockImplementation();
+
+describe('userEvent.longPress with real timers', () => {
```

Comment:
Pls add tests using `test-utils/events` so that we can verify the whole sequence of events, including their payloads. This could replace the existing `jest.fn()` for both called/not called scenarios, as event name will either be present or absent in the events array.

Reply:
I have added a test that also checks payload events with fake timers. Like I explained in https://github.com/callstack/react-native-testing-library/pull/1386#discussion_r1211649723, I don't think it can be tested with real timers